src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.scss

```diff
@@ -6,6 +6,14 @@
 $color-config:    mat.get-color-config($planscape-frontend-theme);
 $primary-palette: map.get($color-config, 'primary');
 
+::ng-deep
+mat-cell:first-of-type {
+  justify-content:left !important;
```

Comment:
I had that originally, but it still would get overridden by the inherent classes. I'll look more into it to see if I missed something though (very much so a newbie with css still 😅)

Reply:
Adding a class (whose name reflects the component this is attached to) is a good idea regardless, as otherwise this rule may end up unintentionally affecting _all_ `mat-cell:first-of-type`s across the entire application.

Reply:
All very good points! Thanks for the wisdom — got it working :)